debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbStreamingChangeEventSource.java

```diff
@@ -256,6 +236,7 @@ else if (oplogStart.getTime() > 0) {
                                 event.getNamespace().getDatabaseName(),
                                 event.getNamespace().getCollectionName());
 
+                        // TODO: Should this be removed as part of DBZ-5102?
```

Comment:
Same as with database filters.. it's part of the pipeline now.

Reply:
Removed.